package.json

```diff
@@ -25,22 +25,22 @@
   },
   "prettier": "@astrojs/site-kit/prettier",
   "dependencies": {
-    "@astrojs/netlify": "^2.2.2",
-    "@fontsource/inter": "^4.5.15",
+    "@astrojs/vercel": "^5.0.1",
+    "@fontsource-variable/inter": "^5.0.8",
+    "@fontsource/inter": "^5.0.8",
     "title": "^3.5.3"
   },
   "devDependencies": {
-    "@astrojs/site-kit": "github:withastro/site-kit",
-    "@astrojs/tailwind": "^3.1.1",
-    "@netlify/functions": "^1.4.0",
+    "@astrojs/site-kit": "github:withastro/site-kit#v5.0.0",
```

Comment:
This actually shouldn't be version locked 😅 the 5.0.0 tag is old. The lockfile ensures this doesn't get randomly upgraded unless we explicitly want that

Reply:
ah ha, ok that also explains all the random headaches I was having here with linting 😅 A day is never complete until I make at least one commit complaining about linters/formatters